### PR TITLE
Feature/nova update new

### PR DESCRIPTION
**Pull request by ghbm-itk**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+- Department and Caseworker classes added.
+- Ability to set security unit, department and caseworker on cases.
+- Better tests for cases.
+
+### Fixed
+
+- Security unit not set properly on Nova cases.
+
+
```
Comment (rimi-itk) [inline changelog.md:22]: https://github.com/DavidAnson/markdownlint and I agree that this superfluous blank line should be removed:

```suggestion
```

File: itk_dev_shared_components/kmd_nova/nova_tasks.py [hunk 2]
```diff
@@ -102,6 +101,25 @@ def get_tasks(case_uuid: str, nova_access: NovaAccess, limit: int = 100) -> list
...
+
+    Args:
+        case_dict: The dictionary describing the task.
+
+    Returns:
+        A case worker object describing the case worker if any.
+    """
+    if 'caseWorker' in task_dict:
+        return Caseworker(
+            uuid = task_dict['caseWorker']['id'],
```
Comment (rimi-itk) [inline itk_dev_shared_components/kmd_nova/nova_tasks.py:115]: You're getting a `uuid` from an `id`. I suppose it is the correct way to do it, but it looks a little odd.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good.

File: itk_dev_shared_components/kmd_nova/nova_tasks.py (hunk 2 above)
Comment (ghbm-itk) [reply itk_dev_shared_components/kmd_nova/nova_tasks.py:115]: It is more of an uuid. And I wanted to create a greater distinction between "id" and "ident" which the api uses.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**